v3/src/components/graph/components/graph-inspector.tsx

```diff
@@ -1,44 +1,68 @@
+import { InspectorButton, InspectorMenu, InspectorPanel } from "../../inspector-panel"
 import ScaleDataIcon from "../../../assets/icons/icon-scaleData.svg"
 import HideShowIcon from "../../../assets/icons/icon-hideShow.svg"
 import ValuesIcon from "../../../assets/icons/icon-values.svg"
 import BarChartIcon from "../../../assets/icons/icon-segmented-bar-chart.svg"
 import StylesIcon from "../../../assets/icons/icon-styles.svg"
 import CameraIcon from "../../../assets/icons/icon-camera.svg"
+import { HideShowMenuList } from "./inspector-panel/hide-show-menu-list"
+import { PointFormatPalette } from "./inspector-panel/point-format-panel"
+import { IGraphModel } from "../models/graph-model"
 
 interface IProps {
+  graphModel: IGraphModel
   show: boolean
+  showParentToggles?: boolean
+  setShowParentToggles?: (show: boolean) => void
+  showMeasuresForSelection?: boolean
+  setShowMeasuresForSelection?: (show: boolean) => void
 }
 
-export const GraphInspector = ({ show }: IProps) => {
+export const GraphInspector = ({ graphModel, show, showParentToggles, setShowParentToggles,
+    showMeasuresForSelection, setShowMeasuresForSelection }: IProps) => {
+  const [showFormatPalette, setShowFormatPalette] = useState(false)
+  const closeAllPalettes = () => {
+    setShowFormatPalette(false)
+  }
+  useEffect(()=>{
+    !show && closeAllPalettes()
+  },[show])
   return (show
-    ? <InspectorPanel component="graph">
-        <InspectorButton tooltip={"DG.Inspector.resize.toolTip"} showMoreOptions={false}
-          testId={"graph-resize-button"} >
-          <ScaleDataIcon />
-        </InspectorButton>
-        <InspectorButton tooltip={"DG.Inspector.hideShow.toolTip"} showMoreOptions={true}
-          testId={"graph-hide-show-button"}>
-          <HideShowIcon />
-        </InspectorButton>
-        <InspectorButton tooltip={"DG.Inspector.displayValues.toolTip"} showMoreOptions={true}
-          testId={"graph-display-values-button"}>
-          <ValuesIcon />
-        </InspectorButton>
-        <InspectorButton tooltip={"DG.Inspector.displayConfiguration.toolTip"} showMoreOptions={true}
-          testId={"graph-display-config-button"}>
-          <BarChartIcon />
-        </InspectorButton>
-        <InspectorButton tooltip={"DG.Inspector.displayStyles.toolTip"} showMoreOptions={true}
-          testId={"graph-display-styles-button"}>
-          <StylesIcon />
-        </InspectorButton>
-        <InspectorButton tooltip={"DG.Inspector.makeImage.toolTip"} showMoreOptions={true}
-          testId={"graph-camera-button"}>
-        <CameraIcon />
-        </InspectorButton>
-      </InspectorPanel>
+    ? <>
+        <InspectorPanel component="graph">
+          <InspectorButton tooltip={"DG.Inspector.resize.toolTip"} showMoreOptions={false}
+            testId={"graph-resize-button"} >
+            <ScaleDataIcon />
+          </InspectorButton>
+          <InspectorMenu tooltip={"DG.Inspector.displayStyles.toolTip"}
```

Comment:
I notice that the tooltips are not currently being shown in English.

Reply:
It should be
```suggestion
          <InspectorMenu tooltip={t("DG.Inspector.displayStyles.toolTip")}
```
in each case.

Reply:
Good catch! I forgot to pass it through the translate function